cmd/libs/go2idl/args/args.go

```diff
@@ -61,6 +61,9 @@ type GeneratorArgs struct {
 	// Package path within the source tree.
 	OutputPackagePath string
 
+	// Output file name.
+	OutputFileBaseName string
```

Comment:
It is used subsequently.  I split many steps like this up to make them
easier to review, that's why a lot of them feel unrelated.  See other
comment about relatedness.

On Tue, Jul 5, 2016 at 4:18 AM, Wojciech Tyczynski <notifications@github.com
> wrote:

> In cmd/libs/go2idl/args/args.go
> <https://github.com/kubernetes/kubernetes/pull/25978#discussion_r69546033>
> :
>
> > @@ -61,6 +61,9 @@ type GeneratorArgs struct {
> >  	// Package path within the source tree.
> >  	OutputPackagePath string
> >
> > +	// Output file name.
> > +	OutputFileBaseName string
>
> This seems to be unused - isn't it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25978/files/042b6cd81b59ba43fee9ab62e1801b6681584472..f5328778be5956ec1237a28c8930e3c2500f023a#r69546033>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVDbsSiF7xQEva2fQIDc82gyNUMX7ks5qSj1vgaJpZM4Ijc3K>
> .
>